client/web/src/enterprise/codeintel/dashboard/components/BrainDot.module.scss

```
@@ -19,10 +19,14 @@
     border: 2px solid var(--body-bg);
 }
 
-.braindot-error::after {
-    background-color: var(--red);
+.braindot-success::after {
+    background-color: var(--green);
+}
+
+.braindot-warning::after {
+    background-color: var(--yellow);
```

```suggestion
.braindot-success::after {
    background-color: var(--success);
}

.braindot-warning::after {
    background-color: var(--warning);
```

We should use semantic variables here so they stay in sync with our other indicators (no change in colors right now)